SETUP.md

```diff
@@ -63,7 +63,11 @@ git remote set-url origin https://github.com/$OWNER/$REPO.git
 git remote set-url origin git@github.com:$OWNER/$REPO.git
 ```
 
-Next, you must manually create an empty GitHub repository at <https://github.com/new>.
+Then you need to create an empty GitHub repository. 
+You can do that at <https://github.com/new> or via the  [GitHub command line interface](https://github.com/cli/cli):
+```sh
+$ gh repo create
```

Comment:
we tend not to prefix terminal commands with `$`, since it makes it harder to copy paste the entire code block